tests/myproject/myproject/flows/batch_flow_with_conda.py

```diff
@@ -42,7 +42,7 @@ def d(self):
         self.next(self.e, foreach="items")
 
     @conda(libraries={"sh": "1.14.2"})
-    @batch(queue="job-queue-nesta-metaflow-sandbox")
```

Comment:
This was missed in #21 (the `queue` in a previous step was changed).

A general point, I think we should:
1) avoid hard-coding queues (that aren't specifically required)
2) avoid default `~/.metaflowconfig/config.json` using AWS resources and instead set `METAFLOW_PROFILE=profname` to select `~/.metaflowconfig/config_profname.json` (using `direnv` and setting a project relevant `METAFLOW_PROFILE` in the `.envrc` makes this frictionless)
3) consider having a separate Metaflow profile and cloudformation stack for testing

Reply:
Thanks for catching! 

> A general point, I think we should:
> 
> avoid hard-coding queues (that aren't specifically required)

I think for the sake of tests it's quite useful to be explicit about the queue, as otherwise digging out the default queue could be roundabout (especially for new developers less familiar with the config setup, given that metaflow docs are simultaneously quite lite and dense)

> avoid default ~/.metaflowconfig/config.json using AWS resources and instead set METAFLOW_PROFILE=profname to select ~/.metaflowconfig/config_profname.json (using direnv and setting a project relevant METAFLOW_PROFILE in the .envrc makes this frictionless)

Is there a mechanism to enforce devs to do this? 

> consider having a separate Metaflow profile and cloudformation stack for testing

this should probably considered in tandem with other stacks (`ojd_daps`, `nesta_daps`, other, testing) and whether or not it is desirable to maintain parallel stacks